kong/assets/dashboards/kong_overview.json

```
@@ -1,267 +1,979 @@
 {
+    "author_name": "Datadog",
+    "description": "Using this dashboard, you can get a high-level view of your Kong metrics so you can monitor its performance.\n\nTo learn more about our Kong integration, see:\n\n- [Our official integration documentation](https://docs.datadoghq.com/integrations/kong/?tabs=host).\n\n- [Monitoring Kong Performance Metrics](https://www.datadoghq.com/blog/monitor-kong-datadog/).\n\n\nYou can clone this dashboard, copy and paste widgets from other out-of-the-box dashboards, and create your own visualizations for your custom applications.",
+    "layout_type": "ordered",
+    "template_variables": [
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "scope",
+            "prefix": ""
+        }
+    ],
     "title": "Kong Overview",
```

Since there's an older `Kong Overview` dashboard already, can we rename this to 
```suggestion
    "title": "Kong Overview (OpenMetrics)",
```

---

Hmm, I'm renaming the old one to `Kong Overview [Legacy]`. Do you think I should be renaming the new one as well?